src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -691,6 +692,49 @@ class DevTask extends AbstractFeatureTask {
             }
         }
 
+        @Override
+        public boolean serverFeaturesModified(File configDirectory) {
```

Comment:
Is the only difference between ci.gradle and ci.maven for this method the `serverXmlFile` handling? If so, it would be better to have this method in ci.common with `serverXmlFile` as an extra parameter (or using the `serverXmlFile` var directly in DevUtil).

Reply:
I think the logic around getting the features should still be duplicated in ci.maven and ci.gradle as the `ServerFeatureUtil` object comes from `ServerFeatureSupport` and `AbstractFeatureTask` respectively. However, I agree we should move the common logic for comparing the two feature lists to ci.common.